### PR TITLE
Adds checkstyle MemberName

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**

### CODE REVIEW

**Comment by shobhitagarwal1612** [comment]
@lognaturel This PR is ready for review

**Comment by shobhitagarwal1612** [comment]
@lognaturel Ready for your review

File: collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java [hunk 1]
```diff
@@ -288,7 +288,7 @@ private void saveAndClose() {
 
     private void saveFile(File f) throws FileNotFoundException {
         if (drawView.getWidth() == 0 || drawView.getHeight() == 0) {
-            // apparently on 4.x, the orientation change notification can occur
+            // apparently on 4.x1, the orientation change notification can occur
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: Looks like this got replaced accidentally!

Comment (shobhitagarwal1612) [reply collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: :sweat_smile:

File: collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java [hunk 2]
```diff
@@ -428,28 +428,28 @@ public void onClick(DialogInterface dialog, int which) {
...
-        private File mBackgroundBitmapFile;
-        private float mX;
-        private float mY;
+        private Bitmap bitmap;
+        private Canvas canvas;
+        private Path currentPath;
+        private Path offscreenPath; // Adjusted for position of the bitmap in the view
+        private Paint bitmapPaint;
+        private File backgroundBitmapFile;
+        private float x1;
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: I'd tend to go for something like `xCoordinate` here. `x1` implies there's more than one.

File: collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java [hunk 3]
```diff
@@ -499,46 +499,46 @@ protected void onDraw(Canvas canvas) {
...
+            currentPath.moveTo(x, y);
 
-            mOffscreenPath.reset();
-            mOffscreenPath.moveTo(x - getBitmapLeft(), y - getBitmapTop());
+            offscreenPath.reset();
+            offscreenPath.moveTo(x - getBitmapLeft(), y - getBitmapTop());
 
-            mX = x;
-            mY = y;
+            this.x1 = x;
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: I think we should only use `this` if there actually is a name conflict.

File: collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java (hunk 2 above)
Comment (lognaturel) [reply collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: Or what exactly does it represent? I guess it's something like `latestX` and `latestY`? The latest point that the user was touching?

File: collect_app/src/main/java/org/odk/collect/android/spatial/GoogleMapsMapBoxOfflineTileProvider.java [hunk 4]
```diff
@@ -132,20 +132,22 @@ private void calculateZoomConstraints() {
             String[] projection = new String[]{"value"};
             String[] minArgs = new String[]{"minzoom"};
 
-            Cursor c = this.mDatabase.query("metadata", projection, "name = ?", minArgs, null, null, null);
+            Cursor c;
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/spatial/GoogleMapsMapBoxOfflineTileProvider.java]: Why was this change made?

File: collect_app/src/main/java/org/odk/collect/android/tasks/DiskSyncTask.java [hunk 5]
```diff
@@ -49,15 +49,25 @@
...
 
-    DiskSyncListener mListener;
+    DiskSyncListener listener;
 
     String statusMessage;
 
-    private FormsDao mFormsDao;
+    private FormsDao formsDao;
+   
+    private static class UriFile {
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/tasks/DiskSyncTask.java]: Is there a convention for private inner classes? I thought they went at the end.

**Review by lognaturel — COMMENTED** [review]
Just a couple things to suggest.

@yanokwa @nribeka This touches EVERYTHING. I want to double triple check that it's really what we want to do. (I do think so, this is a saner convention and I like bringing everything to a consistent state but would like confirmation)

**Comment by yanokwa** [comment]
We need a checkstyle and have agreed on one. There is no good time to do this, so let's do it.

File: collect_app/src/main/java/org/odk/collect/android/tasks/DiskSyncTask.java (hunk 5 above)
Comment (shobhitagarwal1612) [reply collect_app/src/main/java/org/odk/collect/android/tasks/DiskSyncTask.java]: Yes, you are correct. Done.

File: collect_app/src/main/java/org/odk/collect/android/spatial/GoogleMapsMapBoxOfflineTileProvider.java (hunk 4 above)
Comment (shobhitagarwal1612) [reply collect_app/src/main/java/org/odk/collect/android/spatial/GoogleMapsMapBoxOfflineTileProvider.java]: I don't remember that. So I have reverted back the change

File: collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java (hunk 3 above)
Comment (shobhitagarwal1612) [reply collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: Done

File: collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java (hunk 2 above)
Comment (shobhitagarwal1612) [reply collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: xCoordinate can't be used as it does not follow the naming convention. So using latestX and latestY would be better

Comment (lognaturel) [reply collect_app/src/main/java/org/odk/collect/android/activities/DrawActivity.java]: Right, of course. `latestX` and `latestY` sounds good to me.

**Pull request merged**